src/main/java/pages/LandingPage.java

```diff
@@ -0,0 +1,59 @@
+    package pages;
+
+    import org.openqa.selenium.By;
+    import org.openqa.selenium.WebDriver;
+    import org.openqa.selenium.interactions.Action;
+    import org.openqa.selenium.interactions.Actions;
+
+    public class LandingPage {
+
+
+        WebDriver driver;
+        private By addToCartsButton2 = By.cssSelector("a[title='Add to cart']");
+        private By continueShopping = By.cssSelector("span[title='Continue shopping'] span:nth-child(1)");
+        private By carts = By.cssSelector("a[title='View my shopping cart']");
+        private By carts2 = By.xpath("//b[normalize-space()='Cart']");
```

Comment:
Why do we have 2 in these addToCartsButton2 and carts2 locators. Can we improve these namings?